File: Week 01/PS 1/mario.c
# include <stdio.h>
# include <cs50.h>

int main()
{
    int height, i, j, k, hash = 0;

    // Accepting the appropriate value
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    for (i = 0; i < height; i++)
    {
        for (j = i; j < height - 1; j++)
        {
            printf(" "); // Adds spaces in the start which decrease in number everytime the loop runs
        }
        for (k = 0; k < hash + 1; k++)
        {
            printf("#"); // Adds hashtags in the end which increase in number everytime the loop runs
        }
        printf("\n"); // Adds a newline to create more rows
        hash++;
    }
}

File: Week 06/PS 6/mario.py
from cs50 import get_int

# Accepting the appropriate value
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

j = height
for i in range(height):
    print(" "*(j-1) + "#"*(i+1))
    j -= 1   # Decrementing j on every iteration
